libraries/adaptive-expressions-ie11/package.json

```diff
@@ -41,7 +41,8 @@
   "scripts": {
     "build": "webpack",
     "clean": "rimraf dist",
-    "lint": "eslint . --ext .js,.ts"
+    "lint": "eslint . --ext .js,.ts",
+    "lint:fix": "eslint . --ext .js,.ts --fix"
```

Comment:
You should be able to do `yarn lint --fix` already, do you think we need an explicit `lint:fix` script?

Reply:
I tried calling that without these and it threw errors about "lint:fix" script not being available in each library

Reply:
oh..maybe. hang on

Reply:
Disregard.  I had tried adding a root `lint:fix` script. But `yarn lint --fix` does work just fine. Amending...